src/test/java/com/netflix/control/clutch/ClutchConfiguratorTest.java

```diff
@@ -49,5 +50,11 @@
         assertThat(sketch.getQuantile(0.99)).isLessThan(76.0);
     }
 
+    @Test public void shouldGetConfig() {
+        ClutchConfigurator configurator = new ClutchConfigurator(new IClutchMetricsRegistry() {}, 1, 2, Observable.interval(1, TimeUnit.DAYS));
+        configurator.getSketch(Clutch.Metric.CPU).update(70.0);
+        configurator.getConfig();
```

Comment:
Any assertion to add?

Reply:
I think basic assert on not null should suffice.